src/views/Pools/components/CakeVaultCard/CakeVaultTokenPairImage.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
 import { TokenPairImage, ImageProps } from '@pancakeswap/uikit'
-import tokens from 'config/constants/tokens'
-import { getAddress } from 'utils/addressHelpers'
+import { coreTokens, getToken } from 'config/constants/tokens'
 
 const CakeVaultTokenPairImage: React.FC<Omit<ImageProps, 'src'>> = (props) => {
-  const primaryTokenSrc = `/images/tokens/${getAddress(tokens.cake.address)}.svg`
+  const primaryTokenSrc = `/images/tokens/${getToken(coreTokens.cake).address}.svg`
```

Comment:
Succinct example of a core difference in how our token usage will ave to change throughout the app